### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Moiman**

### CODE REVIEW

File: CODE_OF_CONDUCT.rst [hunk 1]
```diff
@@ -113,7 +113,7 @@ You can report issues to the
 at networkx-conduct@groups.io.
 
 If your report involves any members of the Council, or if they feel they have
-a conflict of interest in handling it, then they will recuse themselves from
+a conflict of interest in handling it, then they will recurse themselves from
```
Comment (dschult) [inline CODE_OF_CONDUCT.rst:116]: This change should not be made -- it should be `recuse`

File: networkx/algorithms/connectivity/edge_kcomponents.py [hunk 2]
```diff
@@ -239,7 +239,7 @@ def bridge_components(G):
 class EdgeComponentAuxGraph:
     r"""A simple algorithm to find all k-edge-connected components in a graph.
 
-    Constructing the AuxillaryGraph (which may take some time) allows for the
+    Constructing the AuxiliaryGraph (which may take some time) allows for the
```
Comment (dschult) [inline networkx/algorithms/connectivity/edge_kcomponents.py:242]: This should also not be a CamelCase word (which indicates a class)...  Could we add a space and go lowercase on AuxiliaryGraph -> auxiliary graph

**Review by dschult — COMMENTED** [review]
Thanks for this!
I've taken a pass through and found two comments.

I also want to figure out if this PR should change the "blame" traceback history. Will comment when I've got more info.

**Review by paulitapb — APPROVED** [review]
LGTM

**Review by jarrodmillman — APPROVED** [review]

**Pull request merged**